### PR TITLE
JDK 11 preparation and jakarta uptake.

### PR DESCRIPTION
**Pull request by Tomas-Kraus**

### CODE REVIEW

File: NOTICE.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+which is available at http://www.eclipse.org/org/documents/edl-v10.php. This
+Source Code may also be made available under the following Secondary Licenses
+when the conditions for such availability set forth in the Eclipse Public
+License v. 2.0 are satisfied: GNU General Public License, version 2 with the GNU
+Classpath Exception which is available at
+https://www.gnu.org/software/classpath/license.html.
+
+SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause OR GPL-2.0 WITH
+Classpath-exception-2.0
+
```
Comment (Tomas-Kraus) [inline NOTICE.md]: I know it's wrong. We should keep it as it is now. Will be fixed later when https://www.eclipse.org/projects/tools/documentation.php?id=ee4j.metro is fixed.

Comment (lukasj) [reply NOTICE.md]: do you have a pointer to a bug asking for it?

File: pom.xml [hunk 2]
```diff
@@ -303,6 +328,7 @@
                         </goals>
                         <configuration>
                             <instructions>
+                                <Implementation-Vendor>${vendor.name}</Implementation-Vendor>
                                 <Require-Capability>osgi.ee;filter:="(&amp;(osgi.ee=JavaSE)(version&gt;=1.7))"</Require-Capability>
```
Comment (lukasj) [inline pom.xml]: I'd prefer <_noee>true</_noee> here

File: NOTICE.md (hunk 1 above)
Comment (Tomas-Kraus) [reply NOTICE.md]: https://www.eclipse.org/projects/tools/documentation.php?id=ee4j.metro is fixed now, updating NOTES to be OK.

File: pom.xml (hunk 2 above)
Comment (Tomas-Kraus) [reply pom.xml]: Yes, it makes sense. Fixed.

**Comment by Tomas-Kraus** [comment]
Also removed <addDefaultImplementationEntries>true</addDefaultImplementationEntries> because they are not needed. Implementation-Build-Id changed to groupId (to be the same as in other projects) because previous expression was not expanded properly.

**Comment by Tomas-Kraus** [comment]
Manifest now contains theese entries:
```
Manifest-Version: 1.0
Created-By: Apache Maven Bundle Plugin
Built-By: kratz
Build-Jdk: 11.0.1
Bnd-LastModified: 1541152039709
Bundle-Description: Provides a streaming API to access attachments parts
  in a MIME message.
Bundle-DocURL: https://www.eclipse.org
Bundle-License: http://www.eclipse.org/org/documents/edl-v10.php
Bundle-ManifestVersion: 2
Bundle-Name: MIME streaming extension
Bundle-SymbolicName: org.jvnet.mimepull
Bundle-Vendor: Eclipse Foundation
Bundle-Version: 1.10.0.SNAPSHOT
Export-Package: org.jvnet.mimepull;version="1.10.0"
Implementation-Build-Id: org.jvnet.mimepull
Implementation-Vendor: Eclipse Project for JAX-WS
Tool: Bnd-3.5.0.201709291849
```

**Comment by Tomas-Kraus** [comment]
Merging. We already discussed addDefaultImplementationEntries removal in implementation projects yesterday.

**Pull request merged**